client/src/components/App/Content/SignUp/SignUpForm/SignUpForm.tsx

```diff
@@ -402,7 +408,8 @@ const SignUpForm: React.FC<Props> = ({ defaultValues, handleSaveUser, mode }: Pr
 interface Props {
     defaultValues: SignUpUser;
     handleSaveUser?: () => void;
-    mode: FormMode
+    setIsFormValid?: React.Dispatch<React.SetStateAction<boolean>>;
```

Comment:
why is it optional?

Reply:
because this component is also used to display the user information, and in this state (user details view) then the setIsFormValid is not needed

Reply:
then the component name should probably be something else